### PR TITLE
Update index.jsx

### PR DESCRIPTION
**Pull request by lingdeyiyicifang**
make road label hideable, display road label in default.

### CODE REVIEW

File: src/components/RunMap/index.jsx [hunk 1]
```diff
@@ -16,6 +16,10 @@ import RunMarker from './RunMaker';
 import RunMapButtons from './RunMapButtons';
 import styles from './style.module.scss';
 
+//change 'dispaly' to 'hide' if you want to hide the road label;
+const RoadLabelDisplay = 'display';
```
Comment (yihong0618) [inline src/components/RunMap/index.jsx:20]: make it to `bool`

Comment (lingdeyiyicifang) [reply src/components/RunMap/index.jsx]: changed to 'true' and 'false'

Comment (yihong0618) [reply src/components/RunMap/index.jsx]: not that way, you are still using string.

File: src/components/RunMap/index.jsx [hunk 2]
```diff
@@ -16,6 +16,10 @@ import RunMarker from './RunMaker';
 import RunMapButtons from './RunMapButtons';
 import styles from './style.module.scss';
 
+//change 'true' to 'false' if you want to hide the road label;
+const RoadLabelDisplay = true;
+const layerList=['road-label',  'waterway-label',  'natural-line-label',  'natural-point-label',  'water-line-label',  'water-point-label',  'poi-label',  'airport-label',  'settlement-subdivision-label',  'settlement-label',  'state-label',  'country-label'];
```
Comment (yihong0618) [inline src/components/RunMap/index.jsx:21]: move this to config file like we did maybe better

File: src/components/RunMap/index.jsx [hunk 3]
```diff
@@ -34,6 +38,13 @@ const RunMap = ({
         const map = ref.getMap();
         if (map && IS_CHINESE) {
           map.addControl(new MapboxLanguage({ defaultLanguage: 'zh-Hans' }));
+          if (RoadLabelDisplay == false) {
```
Comment (yihong0618) [inline src/components/RunMap/index.jsx:41]: if (!RoadLabelDisplay)

File: src/components/RunMap/index.jsx [hunk 4]
```diff
@@ -34,6 +38,13 @@ const RunMap = ({
         const map = ref.getMap();
         if (map && IS_CHINESE) {
           map.addControl(new MapboxLanguage({ defaultLanguage: 'zh-Hans' }));
+          if (RoadLabelDisplay == false) { 
+            // todo delete layers
+            map.on('load', () => {
+              for(let layerId of layerList){
+                map.removeLayer(layerId);
```
Comment (yihong0618) [inline src/components/RunMap/index.jsx:45]: can we use layerList.forEach((layerId) => {map.removeLayer(layerId)})

Comment (lingdeyiyicifang) [reply src/components/RunMap/index.jsx]: all done :grin:

File: src/components/RunMap/index.jsx [hunk 5]
```diff
@@ -38,12 +36,10 @@ const RunMap = ({
...
-          if (RoadLabelDisplay == false) { 
-            // todo delete layers
-            map.on('load', () => {
-              for(let layerId of layerList){
-                map.removeLayer(layerId);
-              }
+          if (!ROAD_LABEL_DISPLAY) { 
+          // todo delete layers
+          map.on('load', () => {
+              MAP_LAYER_LIST.forEach((layerId) => {map.removeLayer(layerId)})
```
Comment (yihong0618) [inline src/components/RunMap/index.jsx:42]: add two spaces for this lines

Comment (lingdeyiyicifang) [reply src/components/RunMap/index.jsx]: make it looks better and it works fine, but i don't it's right or not

**Review by yihong0618 — APPROVED** [review]
LGTM can you test in your local env?

**Comment by lingdeyiyicifang** [comment]
> LGTM can you test in your local env?

i tested both locally and they both worked fine, now i run the pulled version in vercel for two days it seems ok too

**Comment by yihong0618** [comment]
OK will merge tonight or tomorrow after test

**Comment by yihong0618** [comment]
@lingdeyiyicifang thanks a lot

**Pull request merged**